src/cards/ha-media_player-card.html

```diff
@@ -248,8 +248,20 @@
 
   playerObjChanged: function (playerObj) {
     var picture = playerObj.stateObj.attributes.entity_picture;
+    var dummy;
     if (picture) {
-      this.$.cover.style.backgroundImage = 'url(' + picture + ')';
+      dummy = document.createElement('IMG');
```

Comment:
@baloob The image will be cached, so I don't think if is needed here.
@armills What is the advantage of iron-image? It will attached to the dom so might (not sure) affect performance more.
I don't think the code will be more compact.